tests/dags/providers/provider_api_scripts/test_inaturalist.py

```diff
@@ -0,0 +1,114 @@
+import logging
```

Comment:
These tests are fantastic!!

Reply:
Thanks. Figuring out the best way to mock s3 has been one of the most difficult parts of this work!